src/jennifer/config.cr

```diff
@@ -77,11 +77,31 @@ module Jennifer
         raise Jennifer::InvalidConfig.new("No database configured") if db.empty?
       end
 
+      def connection_string(*options)
```

Comment:
Makes sense for this to be a responsibility of the Config, this way adapter base doesn't need to reach out to get the URI_PARAMS